nemo/collections/nlp/modules/common/megatron/attention.py

```diff
@@ -25,12 +25,17 @@
 )
 from nemo.collections.nlp.modules.common.megatron.fused_softmax import MatchedScaleMaskSoftmax
 from nemo.collections.nlp.modules.common.megatron.module import MegatronModule
-from nemo.collections.nlp.modules.common.megatron.rotary_pos_embedding import apply_rotary_pos_emb
+from nemo.collections.nlp.modules.common.megatron.position_embedding import XPOSPositionEmbedding
+from nemo.collections.nlp.modules.common.megatron.position_embedding.rotary_position_embedding import (
+    apply_rotary_pos_emb,
+)
 from nemo.collections.nlp.modules.common.megatron.utils import ApexGuardDefaults, attention_mask_func
 from nemo.collections.nlp.parts import utils_funcs
 from nemo.core import adapter_mixins
+from nemo.utils import logging
 
 try:
+    from apex._autocast_utils import _cast_if_autocast_enabled
```

Comment:
@trias702 just added an implementation of this, should we use that one instead since it is more robust?

Reply:
Removed and use `_cast_if_autocast_enabled` from `nemo.collections.nlp.modules.common.megatron.utils `